src/golang/lib/engine/aq_engine.go

```diff
@@ -362,6 +371,8 @@ func (eng *aqEngine) ExecuteWorkflow(
 		execState.Timestamps.FinishedAt = &now
 	}
 
+	eng.deleteTemporaryArtifactContents(ctx, dag)
```

Comment:
hmm this should be in a defer() at the beginning of the method right? Do we not want to delete artifacts when the workflow fails?